core/service/round.ts

```diff
@@ -21,6 +21,8 @@ import { Block } from 'shared/model/block';
 
 const MAX_LATENESS_FORGE_TIME = 500;
 const constants = Config.constants;
+const BLOCK_GENERATE = 'BLOCK_GENERATE';
```

Comment:
Create new file actionTypes.ts for this constants. End create Enum for actions on this file.